runtime/src/main/java/org/corfudb/runtime/view/SealServersHelper.java

```diff
@@ -79,21 +81,23 @@ public static void waitForChainSegmentSeal(Layout.LayoutSegment layoutSegment, M
         for (Layout.LayoutStripe layoutStripe : layoutSegment.getStripes()) {
             CompletableFuture<Boolean>[] completableFutures =
                     completableFutureMap.entrySet().stream()
-                    .filter(pair -> layoutStripe.getLogServers().contains(pair.getKey()))
-                    .map(pair -> pair.getValue())
-                    .toArray(CompletableFuture[]::new);
+                            .filter(pair -> layoutStripe.getLogServers().contains(pair.getKey()))
+                            .map(pair -> pair.getValue())
+                            .toArray(CompletableFuture[]::new);
             QuorumFuturesFactory.CompositeFuture<Boolean> quorumFuture =
                     QuorumFuturesFactory.getFirstWinsFuture(Boolean::compareTo, completableFutures);
+
             boolean success = false;
             try {
-                success = quorumFuture.get();
-            } catch (InterruptedException ie) {
-                throw new UnrecoverableCorfuInterruptedError("Sealing interrupted", ie);
-            } catch (ExecutionException e) {
-                if (e.getCause() instanceof QuorumUnreachableException) {
+                success = CFUtils.getUninterruptibly(quorumFuture);
```

Comment:
CFUtils.getUninterruptibly(quorumfuture, Timeout exception.class, QuorumUnreachableException.class). Then you don't need to catch runtimeException.

Reply:
Done.